test/Microsoft.ML.Tests/TrainerEstimators/TreeEstimators.cs

```diff
@@ -216,8 +221,9 @@ public void TweedieRegressorEstimator()
         public void FastForestRegressorEstimator()
         {
             var dataView = GetRegressionPipeline();
-            var trainer = ML.Regression.Trainers.FastForest( 
-                new FastForestRegression.Options { 
+            var trainer = ML.Regression.Trainers.FastForest(
+                new FastForestRegression.Options
+                {
```

Comment:
nit: Usually I'd pack cleanup for a different section in a different commit / PR.

Reply:
makes sense.. i think i did a control + k + d  on this file  which did some of the cleanups

---
In reply to: [251538653](https://github.com/dotnet/machinelearning/pull/2261#discussion_r251538653) [](ancestors = 251538653)